configuration/micro_kernel_trait.rst

```diff
@@ -34,19 +34,12 @@ Next, create an ``index.php`` file that defines the kernel class and runs it:
         use Symfony\Component\HttpKernel\Kernel as BaseKernel;
         use Symfony\Component\Routing\Attribute\Route;
 
-        require __DIR__.'/vendor/autoload.php';
+        require_once dirname(__DIR__).'/vendor/autoload_runtime.php';
```

Comment:
Don't forget to update the `composer require` section with `symfony/runtime` libs ;)

Reply:
In that section only `symfony/framework-bundle` is required actually, the others are transitive dependencies anyway, not sure we wanted to make them explicit.

Reply:
Good catch! I think that, generally speaking, we don't list transitive dependencies in the docs, so let's remove anything that is not strictly needed. Thanks!